R/save.R

```diff
@@ -32,6 +32,10 @@ saveviz <- function(x, ...) {
 #' @importFrom RColorBrewer brewer.pal
 #' @rdname plot_carp
 #' @export
+
+frame_rate <- 10
+frames_number <- 100
+
```

Comment:
These should be arguments to the `saveviz` method, not global variables, so that users can change them. (As written, there's no way for a user to change these values)

Also, I think 10 fps is a bit slow. What are the current (default) values?